stixcore/products/product.py

```diff
@@ -38,6 +39,7 @@
     32: np.uint32,
     64: np.uint64
 }
+DURATION_FIX_DATE = datetime(2021, 12, 9, 13, 0, tzinfo=timezone.utc)
 
```

Comment:
still needed?

Reply:
yea I think my plan was to replace the datetime below I will do that now so would only be created once and then referenced